src/main/java/de/craften/plugins/rpgplus/components/commands/CustomCommands.java

```diff
@@ -34,6 +36,18 @@ public void registerCommand(String[] commandPath, CommandHandler handler) {
         dispatcher.registerCommand(Arrays.asList(commandPath), handler);
     }
 
+    public void unregisterCommand(String command) {
+    	try {
+            final Field f = getCommandMap().getClass().getDeclaredField("knownCommands");
+            f.setAccessible(true);
+            Map<String, Command> cmds = (Map<String, Command>) f.get(getCommandMap());
+            cmds.remove(command);
+            f.set(getCommandMap(), cmds);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
For future debugging, this should be changed to `throw new UnsupportedOperationException("Could not unregister command " + command, e);`
This will make our life easier when this breaks in a future spigot version.